governor/src/state/keyed.rs

```diff
@@ -98,6 +98,15 @@ where
         )
     }
 
+    pub fn peek_key(&self, key: &K) -> Result<MW::PositiveOutcome, MW::NegativeOutcome> {
```

Comment:
Please add a documentation comment on this public function.

Reply:
Docstring added. Docs are often a matter of taste. Please let me know if you want me to change anything.